<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoveSearchWindow</class>
 <widget class="QMainWindow" name="RemoveSearchWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget
{
	background-color: #FDD935;
}

QLabel
{
font: 17pt &quot;Cooper Black&quot;;
}

QPushButton
{
	font: 15pt &quot;Cooper Black&quot;;
	background-color: #FDD905 ;
	border-Radius: 7px;
	border: 2px solid  black;
}

QPushButton:hover{
color: #FDD905;
	background-color: rgb(0, 0, 0);
}
QLineEdit
{
	font: 19pt &quot;MV Boli&quot;;
}</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>641</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="Name_label">
       <property name="styleSheet">
        <string notr="true">font: 30pt &quot;Cooper Black&quot;;
color: rgb(255, 72, 17);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Genre:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Genre_label">
         <property name="styleSheet">
          <string notr="true">font: 22pt &quot;MV Boli&quot;;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Director:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Director_label">
         <property name="styleSheet">
          <string notr="true">font: 22pt &quot;MV Boli&quot;;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Release Date:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="RDate_label">
         <property name="styleSheet">
          <string notr="true">font: 22pt &quot;MV Boli&quot;;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>125</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Actors:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Actors_label">
         <property name="styleSheet">
          <string notr="true">font: 22pt &quot;MV Boli&quot;;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Playing Date In Cinema:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="Day_label">
         <property name="styleSheet">
          <string notr="true">font: 22pt &quot;MV Boli&quot;;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Time_label">
         <property name="styleSheet">
          <string notr="true">font: 22pt &quot;MV Boli&quot;;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Ticket Numbers:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="TicketEntry">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="topMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QPushButton" name="BackButton">
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="RemoveButton">
         <property name="text">
          <string>Remove  Movie</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
